packages/varo/.changeset/yellow-papayas-perform.md

```diff
@@ -0,0 +1,6 @@
+---
+'@openfn/language-varo': minor
+---
+
+Added components to support converting collection data into Varo message
```

Comment:
You could just copy and paste this into the root `.changesets` folder and delete everything else :)